api/src/main/java/com/talentpath/amazin/api/qAndA/QAndA.java

```diff
@@ -0,0 +1,64 @@
+package com.talentpath.amazin.api.qAndA;
+import java.util.ArrayList;
+
+public class QAndA {
+    private Long id;
+    private String otherIds;
+
+    public QAndA(){
+
+    }
+
+    public QAndA(long id, String otherIds){
+        super();
+        this.id = id;
+        this.otherIds = otherIds;
```

Comment:
I would change otherIds to two things, questions and answers, maybe also upvotes